src/pages/Landing/index.tsx

```diff
@@ -47,8 +47,10 @@ export default () => {
 
   const onSubmit = (e: FormEvent<Element>) => {
     e.preventDefault();
+    addNewCustomer(newUser as Customer)
     setNewUser({ isCool: false });
     setIsModalOpen(false);
+
```

Comment:
You still need to invalidate the customer list in react-query.  Take a look here: https://tanstack.com/query/v4/docs/react/guides/query-invalidation